src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/ApplicationSummaryViewModel.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace SFA.DAS.ApplyService.Web.ViewModels.Roatp
+{
+    public class ApplicationSummaryViewModel
+    {
+        public Guid ApplicationId { get; set; }
+        public string ApplicationReference { get; set; }
+        public string UKPRN { get; set; }
+        public string OrganisationName { get; set; }
+        public string TradingName { get; set; }
+        public string ApplicationRouteId { get; set; }
+
+        public string ApplicationRouteShortText
+        {
+            get
+            {
+                switch (ApplicationRouteId)
+                {
+                    case "1":
+                        {
+                            return "Main";
+                        }
+                    case "2":
+                        {
+                            return "Employer";
+                        }
+                    case "3":
+                        {
+                            return "Supporting";
+                        }
+                }
```

Comment:
There's no default case

Reply:
Fixed, in case we add new routes in the future

Reply:
it's still dropping out of the case statement and returning String.Empty ...